web/modules/custom/dbcdk_community/src/Plugin/Block/FlaggedContentDetails.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\dbcdk_community\Plugin\Block\FlaggedContentDetails.
+ */
+
+namespace Drupal\dbcdk_community\Plugin\Block;
+
+use DBCDK\CommunityServices\Api\ProfileApi;
+use DBCDK\CommunityServices\ApiException;
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Link;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Url;
+use Drupal\dbcdk_community\Url\ModelUrlGenerator;
+use Drupal\dbcdk_community\Url\PropertyUrlGenerator;
+use Drupal\dbcdk_community\Url\UrlGeneratorInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\dbcdk_community\Content\FlaggableContentRepository;
+
+/**
+ * Provides a 'FlaggedContentDetails' block.
+ *
+ * @Block(
+ *  id = "flagged_content_details",
+ *  admin_label = @Translation("Flagged content details"),
+ *  context = {
+ *    "flag_id" = @ContextDefinition("integer")
+ *  }
+ * )
+ */
+class FlaggedContentDetails extends BlockBase implements ContainerFactoryPluginInterface {
```

Comment:
It seems like the naming convention in Drupal suggests that blocks end with Block in their names. What do you think about it?